tests/subsys/bootloader/bl_validation/src/main.c

```diff
@@ -11,6 +11,7 @@
 #include <sys/util.h>
 #include <nrfx_nvmc.h>
 #include <linker/linker-defs.h>
+#include <devicetree_unfixed.h>
```

Comment:
Is there an issue with using `devicetree.h` or do we need the `unfixed` version?